### PR TITLE
Raise friendly error message when issue type can’t be retrieved for a…

### PR DESCRIPTION
**Pull request by sibartlett**
… project

Refs #SUP-30

### CODE REVIEW

File: sdetools/modules/sync_jira/jira_rest.py [hunk 1]
```diff
@@ -93,6 +93,10 @@ def setup_fields(self, issue_type_id, new_issue_template):
             raise AlmException('Could not retrieve fields for JIRA project: %s. %s' %
                                (self.config['alm_project'], error))
 
+        if not meta_info['projects']:
+            raise AlmException('Do not have permission to access issue type "%s" for JIRA project: %s.' %
```
Comment (geoffwhittington) [inline sdetools/modules/sync_jira/jira_rest.py]: `You do not have ....` or `Permission error accessing issue type ....` ?

Comment (geoffwhittington) [inline sdetools/modules/sync_jira/jira_rest.py]: https://developer.atlassian.com/static/rest/jira/6.1.html#d2e1328

"....Projects will not be returned if the user does not have permission to create issues in that project." << That's what we're seeing

Comment (sibartlett) [inline sdetools/modules/sync_jira/jira_rest.py]: Updated error message.

**Comment by geoffwhittington** [comment]
+1

**Pull request merged**